build.sbt

```diff
@@ -7,7 +7,7 @@ name := "spark-corenlp"
 
 version := "0.3.0-SNAPSHOT"
 
-crossScalaVersions := Seq("2.11.8", "2.10.6")
+scalaVersion := "2.11.12"
```

Comment:
Spark doesn't support 2.11.12 yet. See https://issues.apache.org/jira/browse/SPARK-24418. We should use 2.11.8 until 2.4 is released.